src/routes/Profile/helpers.ts

```diff
@@ -208,3 +209,32 @@ export const getClaimButtonModesForVariants = (
 
 export const getSeasonUserId = (userId: string, seasonId: Season['id']) =>
   `${userId.toLowerCase()}#${seasonId}`;
+
+export const getSDKInitParams = (signer: Signer | null) => {
+    const ignoredWalletIds = process.env.REACT_APP_IGNORED_LEAGUE_WALLETS &&
+      process.env.REACT_APP_IGNORED_LEAGUE_WALLETS !== 'UNPROVIDED'
+        ? process.env.REACT_APP_IGNORED_LEAGUE_WALLETS.split(',')
+            .map((s) => s.trim().toLowerCase())
+            .reduce((pV, cI) => ({ ...pV, [cI]: true }), {})
+        : {};
+    if (!process.env.REACT_APP_COMMUNITY_SBT_ADDRESS ||
```

Comment:
move this on top of function. No need to do the complex reduce if this if fails

Reply:
I removed this check, SDK will return the default answer is params are missing ce1158a161687e6b84a00e7a94627b53bfe1607c